Framework/TestHelpers/src/InstrumentCreationHelper.cpp

```diff
@@ -0,0 +1,91 @@
+#include "MantidTestHelpers/InstrumentCreationHelper.h"
```

Comment:
Now that you've created this missing helper. I think the simple instruments such as [this](https://github.com/mantidproject/mantid/blob/15294_geometry_info/Framework/TestHelpers/src/WorkspaceCreationHelper.cpp#L460-L485) can also eventually live there.